### ISSUE TITLE
AttributeError: 'Ledger_KeyStore' object has no attribute 'get_master_private_key'

### ISSUE THREAD

**Issue by ProfessorVenkman** [fyookball/electrum#639]
Crash Report
============

This crash report was reported through the automatic crash reporting system 🤖

Traceback
--------------

```Python traceback
  File "/usr/local/ElectronCash/plugins/cosigner_pool/qt.py", line 206, in on_receive
    xprv = wallet.keystore.get_master_private_key(password)

AttributeError: 'Ledger_KeyStore' object has no attribute 'get_master_private_key'
```

Reporter
------------

This issue was reported by 2 user(s):

| Electrum Version  | Operating System  | Wallet Type  | Locale |
|---|---|---|---|
| 3.1.6  | Linux-4.15.15-300.fc27.x86_64-x86_64-with-fedora-27-Twenty_Seven | 2of2 | da_DK |
| 3.1.6  | Linux-4.15.15-300.fc27.x86_64-x86_64-with-fedora-27-Twenty_Seven | 2of2 | da_DK |

Additional Information
------------------------

The reporting user(s) did not provide additional information.

**Comment by rt121212121** [comment]
Only BIP32 keystore implements this function, but both the commands and the cosigner plugin call it.

**Comment by rt121212121** [comment]
Fixed in commit 67bec77a9512a5e6280d937635cb140996ccc615

**Comment by marceloneil** [comment]
I think most hardware wallets can support message encryption/decryption, or at least some... I'll open a new issue for that later
